File: core/ParamsAndDtos/CandidateParams.cs
using System;

namespace core.ParamsAndDtos
{
    public class CandidateParams: ParamPages
    {
        public int? CandidateId { get; set; }
        public string AppUserId {get; set;}
        public string Email {get; set;}
        public string MobileNo {get; set;}
        public int? ProfessionId {get; set;}
        public DateTime? RegisteredDate { get; set; }
        public DateTime? RegisteredUpto {get; set;}
        
    }
}

File: client/src/app/shared/models/candidateParams.ts
export class candidateParams {
     candidateId?: number;
     applicationNoFrom?: number;
     applicationNoUpto?: number;
     registeredFrom?: Date;
     registeredUpto?: number;
     professionId? = 0;
     agentId?: number;
     industryId?: number;
     associateId?: number;
     appUserId?: number;
     city ='';
     disrict: string;
     state: string;
     email: '';
     mobile: '';

     passportNo?: string;

     sort = "name";
     pageNumber = 1;
     pageSize = 10;
     search: string;
}

